static/js/components/UpdateSourceRedshift.jsx

```diff
@@ -77,7 +78,7 @@ const UpdateSourceRedshift = ({ source }) => {
         }}
         style={{ position: "fixed" }}
       >
-        <DialogTitle>Update Redshift</DialogTitle>
+        <DialogTitle>Update Redshift (or Nullify)</DialogTitle>
```

Comment:
Have we decided on "null" as opposed to "clear" the redshift? 
If not, then the title of the dialog should change as well. 
If "null" then it is ok.